docs/source/clone.rst

```diff
@@ -52,24 +55,13 @@ Clone workflow and component repositories
 Workflow
 ********
 
-There are several ways to clone repositories from GitHub. Below we describe how to clone the global-workflow using either the ssh or https methods. **The ssh method is highly preferred and recommended.**
-
-ssh method (using a password protected SSH key):
-
-::
-
-   git clone git@github.com:NOAA-EMC/global-workflow.git
-
-.. note::
-   When using ssh methods you need to make sure that your GitHub account is configured for the computer from which you are accessing the repository (See `this link <https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account>`_)
-
```

Comment:
Why were instructions for SSH removed? I see that the rest of the documentation doesn't match this recommendation, but I still think SSH should be the preferred method.

Reply:
1. we don't and shouldn't be providing `git` best practices and training in the global-workflow.
2. `https` maximizes accessibility and ease of use.  Is it NOAA policy to enforce SSH authentication on github?

Can you point out which part of documentation does not match this recommendation?  I would like to remove all suggestions of ssh-based references from global-workflow.

Reply:
I meant it was inconsistent with the prior recommendation to use SSH. I think all the examples use https.